### ISSUE TITLE
Autocomplete is showing fragments of current ns in console

### ISSUE THREAD

**Issue by mauricioszabo** [mauricioszabo/atom-chlorine#133]
In a CLJS file, a nil line shows up in the REPL for each character I type that starts to autocomplete eg (str/ (js/ or (desktop. The actual autocomplete results seems OK.

```
desktop.devcards=> nil
desktop.devcards=> nil
desktop.devcards=> nil
desktop.devcards=> nil
desktop.devcards=> nil
desktop.day1=> nil
desktop.day1=> nil
desktop.day1=> 
```

Related to: https://github.com/mauricioszabo/atom-chlorine/issues/127

**Comment by mauricioszabo** [comment]
Probably the problem is on Windows and probably on this file: https://github.com/mauricioszabo/repl-tooling/blob/master/src/repl_tooling/repl_client/clojure.cljs#L244

One possible problem is also the async nature of the socket REPL: needs to be reproduced but if the REPL needs to send three `nil\n` commands, it is possible that something like this is sent to the socket output:

```
nilnil


nil
```

**Comment by mauricioszabo** [comment]
@jc776 I just published version 0.4.4 of Chlorine. Please, test it again, but before you connect to ClojureScript, go to Settings and check "Enable experimental features" please?

I changed a lot the way ClojureScript is evaluated, so it should be fixed. Other problems can occur, though, and the code is quite new so it'll be behind experimental features for a while. Any problems, please open an issue :)

Thanks!

**Comment by jc776** [comment]
Looks good to me - with 0.4.4 + 'Enable experimental' I don't get the extra lines in any of these cases:
`(str/` autocompletes `str/blank?` etc in `day1`
`(str/` doesn't autocomplete in ns that doesn't import it
`(desktop.day1/` autocompletes `desktop.day1/fuel-examples` etc
`(mod` autocompletes `module-fuel` in `day1`
`(js/` doesn't autocomplete
`(desktop.` doesn't autocomplete (until full ns name is entered above)
(same results without prefix `(`)

**Comment by mauricioszabo** [comment]
Great, glad to know its working.

There's also support now for clj-suitable, so if you add it on your Shadow-CLJS dependencies (don't worry, your production bundle will not include it in any way - it'll be just a development dependency) you can also complete `js/` variables.

I'm gonna close this issue, if you still have problems please reopen it :)

**Comment by mauricioszabo** [comment]
Fixed on `0.4.4`, with "experimental features" toggled.
